### PR TITLE
[IMP] website_visitor: avoid double indexing

### PR DESCRIPTION
**Pull request by nseinlet**
unique constraint create an btree unbique index:
```
# \d website_visitor
[...]
Indexes:
    "website_visitor_pkey" PRIMARY KEY, btree (id)
    "website_visitor_access_token_index" btree (access_token)
    "website_visitor_access_token_unique" UNIQUE CONSTRAINT, btree (access_token)
    "website_visitor_partner_uniq" UNIQUE CONSTRAINT, btree (partner_id)
```
removing index=True avoid this double indexing.

### CODE REVIEW

File: addons/website/models/website_visitor.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ class WebsiteVisitor(models.Model):
     _order = 'last_connection_datetime DESC'
 
     name = fields.Char('Name')
-    access_token = fields.Char(required=True, default=lambda x: uuid.uuid4().hex, index=True, copy=False, groups='base.group_website_publisher')
+    access_token = fields.Char(required=True, default=lambda x: uuid.uuid4().hex, copy=False, groups='base.group_website_publisher')
```
Comment (odony) [inline addons/website/models/website_visitor.py:34]: I'd keep it explicit to make it obvious that it wasn't just forgotten.

```suggestion
    access_token = fields.Char(required=True, default=lambda x: uuid.uuid4().hex, index=False, copy=False, groups='base.group_website_publisher')
```

**Review by odony — APPROVED** [review]
Might possibly target 13.0 or 14.0, because it's a bugfix and there's no associated risk AFAICS. It will just save new databases a redundant index.

**Comment by nseinlet** [comment]
13.0 PR https://github.com/odoo/odoo/pull/60042

**Comment by odony** [comment]
superseded by #60042

**Pull request closed without merging**